src/kOS.Safe/Compilation/CompiledObject.cs

```diff
@@ -57,6 +58,7 @@ public static void InitTypeData()
             AddTypeData(5, typeof(float));
             AddTypeData(6, typeof(double));
             AddTypeData(7, typeof(string));
+            AddTypeData(8, typeof(KOSArgMarkerType));
```

Comment:
KSM files need to store arguments of the type KOSArgMarkerType because of all the places where we had code that does:
```
   push $<argstart>
```
which used to be pushing a STRING, are now pushing a new type:
```
    push _ARGMARKER_
```
And that opcode argument needs to be storable in KSM.